src/mame/misc/rawthrillspc.cpp

```diff
@@ -117,7 +117,34 @@ ROM_START( fnf )
 	DISK_IMAGE_READONLY( "faf3.06d2", 0, SHA1(183664482f6665adffc74d69e28338da740443c5) )
 ROM_END
 
+/*
+ Doodle Jump Arcade, by default, uses a different PC than other Raw Thrills games:
+   -HP with an Asus M2N68-LA motherboard (socket AM3).
+   -CPU Athlon II CPU (ADX2400CK23GQ).
+   -2GB RAM (single SIMM, PC3-12800) [HP P/N 655409-150].
+   -nVidia Club3D CGNX-G942LI.
+ The game runs over Linux CentOS.
+ I/O board with ICE P/N X2034X.
+ HASP USB security dongle.
```

Comment:
Does this use the same chipset as the other games in this source file?  If it doesn’t, does it really belong with them, or is it a different system altogether?

Reply:
Both are nForce 400 based (https://en.wikipedia.org/wiki/Comparison_of_Nvidia_nForce_chipsets).

[M2N68-LA.pdf](https://github.com/mamedev/mame/files/10686809/M2N68-LA.pdf)

![image](https://user-images.githubusercontent.com/5889052/217564671-f217cc62-2549-46c9-8349-6b19ee33415d.png)